app/javascript/css/main.scss

```diff
@@ -0,0 +1,251 @@
+
+html, body {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  padding: 0;
+}
+
+body {
+  color: #333;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

Comment:
let's use a custom font to display messages
for example https://www.fontsquirrel.com/fonts/Capture-it